### PR TITLE
Niyousha

### PR DESCRIPTION
**Pull request by niyoushasaeidi**

### CODE REVIEW

**Comment by fatemahaz** [comment]
There is an error in stdIdentifiers - should remove the first println

**Comment by ccdu57** [comment]
There is no exclamationExclamation (line 123) and some of those double characters should be renamed, barBar should be doubleOrBar

File: ptsrc/parser/scan.ssl [hunk 1]
```diff
@@ -161,8 +176,36 @@ Scan :
...
+                [
+                    | '&':
+                        .pDoubleAmpersand
+                    | *:
+                        .pAmpersand
+                ]
+            | '|':
+                [
+                    | '|':
+                        .pBarBar
```
Comment (sesinclair) [inline ptsrc/parser/scan.ssl]: I think this needs to be changed to .pOrBar

Comment (sesinclair) [reply ptsrc/parser/scan.ssl]: I think this needs to be changed to .pDoubleOrBar **

File: ptsrc/parser/scan.ssl [hunk 2]
```diff
@@ -161,8 +176,36 @@ Scan :
...
+                    | *:
+                        .pBar
```
Comment (sesinclair) [inline ptsrc/parser/scan.ssl]: I think this needs to be changed to .pOrBar

**Review by sesinclair — CHANGES_REQUESTED** [review]

File: ptsrc/parser/scan.ssl (hunk 2 above)
Comment (niyoushasaeidi) [reply ptsrc/parser/scan.ssl]: It is just a naming convention I can change it

**Comment by ccdu57** [comment]
closing this PR. Merge conflicts have been resolved and pushed to new branch for merge. refer to this https://github.com/fatemahaz/CMPE458/pull/5

**Pull request closed without merging**